projects/storefrontlib/src/shared/components/dialog/index.ts

```diff
@@ -0,0 +1 @@
+export * from './dialog.module';
```

Comment:
if we do agree on a dialog component, we should export it here I assume.

Reply:
Removed